idb/postgres/postgres.go

```diff
@@ -954,18 +954,14 @@ func (db *IndexerDb) yieldTxnsThreadSimple(ctx context.Context, rows pgx.Rows, r
 				}
 			}
 		}
-		select {
-		case <-ctx.Done():
```

Comment:
Removing the context from these functions because `pgx.Rows` is the blocking object and it's responsible for terminating early?

Reply:
Yes, `Rows.Next()` is blocking but will terminate early. `Rows.Err()` will return any error occurred while reading rows, including context expiration.